pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -320,3 +339,30 @@ func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.CronJob
 
 	return cj, err
 }
+
+func (r *Reconciler) UpdateFromLoggingConfigMap(cfg *corev1.ConfigMap) {
```

Comment:
Is this intended to push updated logging/metrics config by restarting all the CronJob receive adapter Pods? Because, as-is, I don't think it will restart anything. I think it will only use the new logging config on the next reconcile, likely at the resync period (10 hours IIRC).

Reply:
True, it wouldn't restart itself. That's how we have done for apiserver source as well.

Reply:
Let's leave it as is for now, and we can update it later on. Because we will have to kill the data plane pod so that it's recreated with the new config

Reply:
Let's add a `// TODO determine how to push the updated logging config to existing data plane Pods.`